Exiled.Loader/Loader.cs

```diff
@@ -134,7 +134,8 @@ static Loader()
         /// <param name="dependencies">The dependencies that could have been loaded by Exiled.Bootstrap.</param>
         public static void Run(Assembly[] dependencies = null)
         {
-            if ((RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)) || geteuid() == 0)
+            if ((RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)) ||
+                ((RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) && geteuid() == 0))
```

Comment:
The two non-windows OS checks are unnecessary. You can simply check if it's windows. If it's windows, check if it's windows root, otherwise check if it's linux root.